Libplanet.Net/Swarm.BlockSync.cs

```diff
@@ -476,26 +477,47 @@ in completedBlocks.WithCancellation(cancellationToken))
                         $"is unusable for the the existing chain #0 {g}.");
                 }
 
+                if (!(tipCandidate is { }))
+                {
+                    throw new SwarmException(
+                        $"Tip candidate is null.");
+                }
+
                 // NOTE: All blocks in deltaBlocks will have non-null previousHash.
-                var deltaBlocks = new LinkedList<Block<T>>(new[] { tipCandidate });
+                var blockChainSlice = new BlockChainSlice(new[] { tipCandidate.Hash });
                 while (true)
                 {
-                    Block<T> b = deltaBlocks.First();
+                    BlockHash bh = blockChainSlice.Last();
+                    if (!workspace.Store.ContainsBlock(bh))
+                    {
+                        break;
+                    }
```

Comment:
Is this check needed? I think that we're checking this with `.GetBlock<T>()` or `GetBlockDigest()`.

Reply:
I forgot to remove this check 😓 I'll remove this check next commit.

Reply:
ec496acdd1366ea75a3fed132e352d5f47354809 amended